app/js/components/orderEntry/OrderEntryPage.jsx

```diff
@@ -27,16 +27,26 @@ export class OrderEntryPage extends React.Component {
         </div>
       );
     }
-    if (settingEncounterType.length === 0 && error) {
+
+    if (settingEncounterType.length === 0 || error) {
       return (
         <div className="error-notice">
           <p>
-            Setting for <strong>order.encounterType</strong> does not exist.
-            Please contact your administrator to create one for you.
+            Configuration for <strong>order.encounterType</strong> {error === 'incomplete config' ? 'is incomplete' : 'does not exist'}.
+            Please contact your administrator for more information.
           </p>
           <p>
-            As an Administrator, if you have already configured this setting, please check
-            if its name corresponds to <strong>order.encounterType</strong>
+            As an Administrator,&nbsp;
+            {
+              error === 'incomplete config' ?
+                <span>
+                  please ensure that you have created a valid <strong>encounter type</strong>.
+                </span> :
+                <span>
+                  ensure that you have created a setting called <strong>order.encounterType</strong>&nbsp;
+                  with a value corresponding to a valid encounter type.
```

Comment:
What do you mean by a valid encounter type?

Reply:
A valid encounter type would refer to a non-empty object defined below
`
{
  "name": "order entry",
  "description": "order entry ui config"
}
`
The idea is to make the error description as generic as possible to protect the structure of the database from the end user, but a little clearer to the admin to be able to set it up (as suggested by Darius [here](https://talk.openmrs.org/t/implementation-of-the-order-entry-ui-application-in-reactjs/16791/35)). A detailed instruction for setting up the configurations is defined in the [Readme](https://github.com/openmrs/openmrs-owa-orderentry/blob/master/README.md) file

Reply:
Would the encounter type value be a name, uuid, or numeric encounterId

Reply:
The encounter type would have to be setup by the admin and that requires making a POST request to the encounterType endpoint with the credentials defined above
<img width="1680" alt="screen shot 2018-05-03 at 10 37 04 pm" src="https://user-images.githubusercontent.com/8871565/39604056-90e6bc66-4f22-11e8-9836-0a6ca329f102.png">
<img width="1680" alt="screen shot 2018-05-03 at 10 38 11 pm" src="https://user-images.githubusercontent.com/8871565/39604093-b0ba63f8-4f22-11e8-9394-f0d2e63e9540.png">